utilities/calibration-delta-updater/README.md

```diff
@@ -0,0 +1,24 @@
+# calibration-delta-updater
+
+**calibration-delta-updater** is a simple tool for updating the calibration delta of the robot.
+
+## :computer: Dependencies
+
+**calibration-delta-updater** depends of the [`YarpImplementation` of the `ISensorBridge`](https://github.com/dic-iit/bipedal-locomotion-framework/tree/master/src/RobotInterface/YarpImplementation) and on the [python bindings](https://github.com/dic-iit/bipedal-locomotion-framework/tree/master/bindings/python/RobotInterface). To run the script you also need to install some additional python dependencies
+
+```
+pip install numpy lxml
```

Comment:
I am a bit scared by this. If the canonical was of installing blf is by using apt dependencies, can't we install numpy and lxml from apt?
```suggestion
sudo apt-get install python3-numpy python3-lxml
```
?